packages/components/src/components/composite-checkbox/composite-checkbox.jsx

```diff
@@ -44,7 +44,7 @@ CompositeCheckbox.propTypes = {
     label: PropTypes.string.isRequired,
     id: PropTypes.string,
     description: PropTypes.string.isRequired,
-    children: PropTypes.oneOfType(PropTypes.node, PropTypes.arrayOf(PropTypes.node)),
+    children: PropTypes.oneOfType([PropTypes.node, PropTypes.arrayOf(PropTypes.node)]),
```

Comment:
This is for fixing this propTypes error:

![image](https://user-images.githubusercontent.com/103577623/182836641-b0d57d6f-31a7-4e77-ad25-68a8b13db27f.png)